src/applications/combined-debt-portal/medical-copays/components/BalanceCard.jsx

```diff
@@ -95,7 +96,17 @@ const BalanceCard = ({ id, amount, facility, city, date }) => {
       >
         {linkText}
         <va-icon icon="navigate_next" size={3} />
-      </Link>
+      </Link> */}
```

Comment:
I think we should continue using Link from `react-router-dom` when navigating within the CDP, it helps prevent reloading the whole app, and making additional api calls